README.md

```diff
@@ -27,12 +27,11 @@ const config = require('./gulp-config');
 const $ = require('./node_modules/webfactory-gulp-preset/plugins')(config); // loads all gulp-* modules in $.* for easy reference
 
 
-const { scripts } = require('./node_modules/webfactory-gulp-preset/tasks/scripts');
-const { styles } = require('./node_modules/webfactory-gulp-preset/tasks/styles');
+const { webpack } = require('./node_modules/webfactory-gulp-preset/tasks/webpack');const { styles } = require('./node_modules/webfactory-gulp-preset/tasks/styles');
 const { browsersync } = require('./node_modules/webfactory-gulp-preset/tasks/browsersync');
 
 function js(cb) {
-    scripts(gulp, $, config);
+    webpack(gulp, $, config);
```

Comment:
Das ist eher ein anderer PR? Ich hatte in der Readme bewusst erst mal den alten Weg behalten (Webpack wird weiter unten erklärt), weil die meisten Projekte noch nicht umgestellt sind/waren.

Können wir auch umdrehen, dann sollte der alte Weg vielleicht unten als Alternative erläutert werden?